apps/fortuna/.gitignore

```diff
@@ -2,3 +2,5 @@
 *config.yaml
 *secret*
 *private-key*
+.envrc
+fortuna.db
```

Comment:
@m30m are these safe to gitignore?  I noticed they weren't here before; I assume it's just an oversight but was there a good reason?